uswds/uswds
# USWDS - Header: Vertically center icon in nav

# Summary

Fixed the vertical alignment of the expand icon on header navigation items.

## Related issue 
Closes #5660 

## Preview link

[Header component](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/feature-vertically-center-nav-elements/?path=/story/components-header--default)

## Related PR
This work originated in https://github.com/uswds/uswds/pull/5267 (Thanks @aduth!)
[Changelog →](https://github.com/uswds/uswds-site/pull/2389)

## Problem statement

When navigation links include text which breaks to multiple lines (particularly in mobile navigation), the expander icon is shown at the bottom right of the link, which makes it more difficult to identify.

## Solution

Vertically center the icon.

Before|After
---|---
![Screen Shot 2023-05-02 at 10 51 40 AM](https://user-images.githubusercontent.com/1779930/235704503-ebbe7c6b-8f7c-440c-98c1-4f5bd6ab1edc.png)|![Screen Shot 2023-05-02 at 10 52 04 AM](https://user-images.githubusercontent.com/1779930/235704519-ee0f6c75-8feb-4086-b9ed-077b082fdd74.png)


## Testing and review

1. Visit header variant pages
2. Inspect mobile styles
3. Create large link span for accordion buttons
4. Confirm expander icon maintains vertical alignment

### Testing checklist

- [ ] Mobile navigation expander icons maintain vertical alignment
- [ ] No visual regression in desktop variants
